docs/declarative-customization/list-form-conditional-show-hide.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Show or hide columns in a list form
+description: Customize which columns to show or hide using a conditional formula in the list form by constructing a simple formula that are equations that perform conditional checks on values in a SharePoint list or library. 
+ms.date: 03/16/2020
+localization_priority: Priority
+---
+
+# Show or hide columns in a list or library form
+You can show or hide columns in a list or library form as an alternative to deleting. When you hide a column, it doesn't affect the column or the data in the column, as it would if you deleted it. To re-use the column, you can just show it again in the form.
```

Comment:
You can show or hide columns in a list or library form as an alternative to deleting them. When you hide a column, it doesn't affect the column or the data in the column, as it would if you deleted it. To re-use the column, you can simply show it again in the form.